src/pages/Regions/AddRegion.tsx

```diff
@@ -108,165 +120,180 @@ const AddNewRegionFormPage = () => {
     setCurrentPhoto(false);
   };
 
+  function seeRegionNameExistsModal() {
```

Comment:
What does it is mean `seeRegionNameExistsModal`?
I guess it should be `showCheckingRegionNameModal`

Reply:
It means to see the modal which says that region name already exists, not the modal which will check if the region name exist. So I could change it to "showRegionNameExistsModal"

Reply:
Done!